src/components/plots/barPlot.ts

```diff
@@ -34,6 +34,12 @@ export module Plot {
       this._animators["baseline"] = new Animator.Null();
       this._isVertical = isVertical;
       this._baselineValue = 0;
+
+      if (isVertical) {
+        Bar._updateDomainerBaseline(yScale, 0, this._plottableID);
+      } else {
+        Bar._updateDomainerBaseline(xScale, 0, this._plottableID);
```

Comment:
Why this change? I understand removing instance calls, but `getID()` should be very safe...